cves/CVE-2013-1965.yml

```diff
@@ -162,8 +176,11 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
```

Comment:
So the vulnerability is still applicable if this feature is enabled?

Reply:
That is correct:  The fix was to disable this feature and give a warning if another module tries to enable it

Reply:
Gotcha. I would add a blurb to the description that mentions that, but that's something you can easily get from context on the fix